.github/workflows/validate_ansible_roles.yaml

```diff
@@ -8,23 +8,28 @@ jobs:
   validate:
     name: Validate Ansible Roles
     runs-on: ubuntu-20.04
+    container: fedora:latest
```

Comment:
Is this change somehow directly related to this PR?

Reply:
For some reason, this PR seemed to fail with the existing ubuntu-20.04 setup. I tracked it down to the complaint of pyYAML, which upstream fedora had later version and this did not fail.
https://github.com/ComplianceAsCode/content/pull/7055/checks?sha=0e2d887d8e8feabf073c2e97cee7c2e788497cae#step:6:28

However, let me double-check this issue and make sure it's not content-related, in which case I can just revert this change.

Reply:
@vojtapolasek Okay - for context reference, there were a couple of issues at play here.. 

1. Having a block in the following order caused issues:
```yaml
    - name: 'Limit Password Reuse: system-auth'
      block:

        - name: 'Limit Password Reuse: system-auth - Ensure correct control value in pam_pwhistory'
          pamd:
            name: system-auth
            type: password
        ...
```

-   It doesn't like the two `- name` stanza's having single quotes
-   I had to remove the `{{{ rule_title }}}` from the second `-name` to get it to work. 

2. There was some sort of discrepancy in the reference section for the `accounts_password_pwhistory_remember_password_auth` rule.yml file that was causing a failure with checking for valid yaml tags in https://github.com/ComplianceAsCode/content/blob/master/utils/ansible_playbook_to_role.py#L322
-  I cleaned up the references and that fixed the issue.

Finally, just a point about using the container approach I suggested - none of the above issues were present, seems like newer versions of PyYAML fixed the above issues. I believe I can get the latest version of PyYAML using pip, I will be submitting a PR with this change as soon as I test it out.

Reply:
> I had to remove the `{{{ rule_title }}}` from the second `-name` to get it to work.

Could it be related to this https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#when-to-quote-variables-a-yaml-gotcha ?

Reply:
Not sure how, when the ansible playbook to role script runs, the `{{{ rule_title }}}` has already been converted, so there would be no gotcha's for yaml at that point. My guess is, maybe instead of:
```yaml
- name: "{{{ rule_title }}}"
  ...
```
I could try removing the double quotes?:
```yaml
- name: {{{ rule_title }}}
  ...
```